projects/pluto/src/main/java/nl/fairspace/pluto/web/dto/MetadataSourceInfo.java

```diff
@@ -0,0 +1,16 @@
+package nl.fairspace.pluto.web.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@EqualsAndHashCode
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
do we need no args constructor if there is no setter?

Reply:
I'll remove it